lms/static/sass/views/_program-list.scss

```diff
@@ -13,6 +13,26 @@
   @include span-columns(12);
 }
 .sidebar{
+    float: right !important;
+    //padding: $baseline;
+    margin-left: 2.35765%;
+    margin-bottom: 20px;
+    width: 23.23176%;
```

Comment:
Please use the span-columns approach used elsewhere in the file instead of adding custom percentage widths.